{
  "schema_version": "1.4.0",
  "id": "GHSA-m2c7-42rf-c62f",
  "modified": "2022-02-14T19:58:19Z",
  "published": "2022-02-01T00:00:44Z",
  "aliases": [
    "CVE-2021-44255"
  ],
  "summary": "Unrestricted Upload of File with Dangerous Type in motionEye",
  "details": "motionEye <= 0.42.1 and motioneEyeOS <= 20200606 allow a remote attacker to upload a configuration backup file containing a malicious python pickle file. This is possible when an installation is accessible over the Internet and uses no or poor authentication credentials.\n\nThe GitHub repositories for motionEye and motionEyeOS are no longer being actively maintained as of January 2022, so release of a patched version is unlikely. Keeping a motionEye or motionEyeOS installation off of the Internet and/or using strong credentials provide protection against this issue.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "motioneye"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "last_affected": "0.42.1"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-44255"
    },
    {
      "type": "WEB",
      "url": "https://github.com/ccrisan/motioneyeos/issues/2843"
    },
    {
      "type": "PACKAGE",
      "url": "https://github.com/ccrisan/motioneye"
    },
    {
      "type": "WEB",
      "url": "https://www.pizzapower.me/2021/10/09/self-hosted-security-part-1-motioneye"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-434"
    ],
    "severity": "HIGH",
    "github_reviewed": true,
    "github_reviewed_at": "2022-02-14T19:58:19Z",
    "nvd_published_at": "2022-01-31T12:15:00Z"
  }
}